en/key_concepts.rst

```diff
@@ -0,0 +1,151 @@
+
+See also the documentation page about `node parameters`_.
+
+Content types
+-------------
+
+Content types are a way to create new kinds of content that will be available to contribution.
+Creating a content type allows to define the fields available in the future contents.
+
+For instance, a 'news' content will have differents fields from an 'article' content,
+so there would be two distinct content types for these objects : one for the news contents and another for the article contents.
+
+See also `Presentation of the Content Types`_.
+
+Contents
+--------
+
+With Open Orchestra, contents are used to display contributed information (articles, news, etc.).
+The contents are defined by a content type.
+Like nodes, contents are multilingual, versionned and have a validation workflow.
+
+See also `Presentation of contents`_.
+
+Templates
+---------
+
+Templates are used to preconfigure pages by setting zones that are commonly used accross pages.
+Open Orchestra let the user start with a template when creating a node.
+
+See also `configuring a template`_.
+
+Zones
+---------
+
+Zones are the way to organize the architecture of the page.
+They are embedded in the nodes and can contain other zones or blocks.
+
+See also `configuring a zone`_.
+
+Blocks
+---------
+
+A block is the base brick representing any visible element on a page.
+Aggregating blocks found in the nodes will lead to the construction of the pages.
+
+See also the `list of blocks`_ available.
+
+Transverses blocks
+------------------
+
+Transverses blocks are defined inside the transverse node, they can be shared by several nodes of a website.
+All blocks can be transverse, as long as they are added to the transverse node.
+Transverse blocks can only be conigured in the transverse node.
+
+Themes
+------
+
+A theme represents the visual identity of a website, they  CSS and JavaScript files
+that work together in order to render the websites in a certain way.
+
+See how to add a `theme`_.
+
+Websites
+--------
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+See also how to configure a `website`_.
+
+Devices
+-------
+
+Open Orchestra is multi-device and is able to display, on the Front Office,
+different templates depending on the device of the visitor.
+
+See also `multi-devices`_.
+
+Keywords
+--------
+
+Keywords can be created in the Back Office, and then be used to tag contents and medias.
+
+Users
+-----
+
+Users represent people that can connect to the Open Orchestra Back Office and make contributions.
+They also are the Front Office users who can access to the private pages of a website.
+It's also possible to assign groups to users.
+
+See also how to configure a `user`_.
+
+Roles
+-----
+
+Roles allow to define authorization in the Back Office.
+
+See also how to create a `role`_.
+
+Groups
+-----------
+
+Groups combine roles and are assigned to users.
```

Comment:
A group can also be link to a website.

You can have different roles when you are editing siteA or siteB